massdriver-application-gcp-cloud-run/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "container_image" {
+  type = string
+}
+
+variable "max_instances" {
+  type    = number
+  default = 5
```

Comment:
what's the value of a default max of 5 here? feels to me like the default max should be effectively infinite maybe bundles can decided to set this value more conservatively for a development guided config?

FWIW I think defaults in our modules are super opaque to end users.

Reply:
This is a great question! I made it 5 because the CR service defaults to 5 if it's unset. 🤷‍♂️  This is a small safety against accidental scale.

Reply:
I would just hate to see this silently get unset in a bundle, we don't notice cuz tf uses the default and then someone is cries in a blog post about how massdriver apps don't scale beyond toy scale.

Reply:
Fair fair. I can yank the default here and bubble it up to the massdriver.yaml